app/controllers/account/email_passwords_controller.rb

```diff
@@ -15,7 +15,7 @@ def update_email
     elsif current_user.update(email: new_email)
       EventLog.record_email_change(current_user, current_email, new_email)
       UserMailer.email_changed_notification(current_user).deliver_later
-      redirect_to root_path, notice: "An email has been sent to #{new_email}. Follow the link in the email to update your address."
```

Comment:
In `email_change_notice` you've changed `#{new_email}` to `#{current_user.unconfirmed_email}`. Presumable these are the same value?